dataplane/cmd/redbeams.go

```diff
@@ -15,32 +15,92 @@ import (
 func init() {
 	DataPlaneCommands = append(DataPlaneCommands, cli.Command{
 		Name:  "redbeams",
-		Usage: "manage RBDMS instances and databases",
+		Usage: "manage relational databases and database servers",
 		Subcommands: []cli.Command{
 			{
 				Name:  "dbserver",
-				Usage: "manage redbeams instances",
+				Usage: "manage redbeams database servers",
 				Subcommands: []cli.Command{
 					{
 						Name:   "list",
-						Usage:  "list all RBDMS instances",
+						Usage:  "list all database servers",
 						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
-						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn, fl.FlWaitOptional).AddAuthenticationFlags().Build(),
-						Action: redbeams.ListRBDMSInstances,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn).AddAuthenticationFlags().AddOutputFlag().Build(),
+						Action: redbeams.ListDatabaseServers,
 						BashComplete: func(c *cli.Context) {
-							for _, f := range fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn, fl.FlWaitOptional).AddAuthenticationFlags().Build() {
+							for _, f := range fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn).AddAuthenticationFlags().AddOutputFlag().Build() {
+								fl.PrintFlagCompletion(f)
+							}
+						},
+					},
+					{
+						Name:   "get",
+						Usage:  "get a database server by CRN",
+						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlCrn).AddAuthenticationFlags().AddOutputFlag().Build(),
+						Action: redbeams.GetDatabaseServer,
+						BashComplete: func(c *cli.Context) {
+							for _, f := range fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlCrn).AddAuthenticationFlags().AddOutputFlag().Build() {
+								fl.PrintFlagCompletion(f)
+							}
+						},
+					},
+					{
+						Name:   "getbyname",
+						Usage:  "get a database server by name",
+						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlEnvironmentCrn, fl.FlName).AddAuthenticationFlags().AddOutputFlag().Build(),
+						Action: redbeams.GetDatabaseServerByName,
+						BashComplete: func(c *cli.Context) {
+							for _, f := range fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentCrn, fl.FlName).AddAuthenticationFlags().AddOutputFlag().Build() {
+								fl.PrintFlagCompletion(f)
+							}
+						},
+					},
+					{
+						Name:   "getstatus",
+						Usage:  "get the status of a database server by CRN",
+						Before: cf.CheckConfigAndCommandFlagsWithoutWorkspace,
+						Flags:  fl.NewFlagBuilder().AddFlags(fl.FlCrn).AddAuthenticationFlags().AddOutputFlag().Build(),
+						Action: redbeams.GetDatabaseServerStatus,
+						BashComplete: func(c *cli.Context) {
+							for _, f := range fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlCrn).AddAuthenticationFlags().AddOutputFlag().Build() {
+								fl.PrintFlagCompletion(f)
+							}
+						},
+					},
+					{
+						Name:   "getstatusbyname",
```

Comment:
I am not sure getStatus is needed, as the resource in response does contain the status already.

Reply:
"status" would be better than "getstatus"

Reply:
Right now, most command names here and elsewhere are verbs: describe, list, delete. So, I like getstatus over just status (a noun).

As for whether this command is needed, we discussed the question this morning, and we are going to rearrange how redbeams works, and its API, so that this command will naturally disappear anyway. 😄